layers/+os/osx/keybindings.el

```diff
@@ -41,24 +41,30 @@
 
   ;; Set internal variables according to the given config variables
   (cl-loop for (key-var . internal-var) in modifier-keys do
-           (let ((key-value (symbol-value key-var)))
-             (when (member key-value allowed-values)
-               (setf (symbol-value internal-var) key-value))))
+    (let ((key-value (symbol-value key-var)))
+      (when (member key-value allowed-values)
+        (setf (symbol-value internal-var) key-value))))
 
   (defun kbd-mac-command (keys)
     "Wraps `kbd' function with Mac OSX compatible Command-key (⌘).
+     (when osx-swap-option-and-command
+        (cl-rotatef mac-command-modifier mac-option-modifier))
+
 KEYS should be a string such as \"f\" which will be turned into values
 such as \"H-f\", \"s-f\", or \"A-f\" depending on the value of
 `mac-commmand-modifier' which could be `hyper', `super', or `alt'.
 KEYS with a string of \"C-f\" are also valid and will be turned into
 values such as \"H-C-f\".
 Returns nil if `mac-command-modifier' is set to `none' or something
 other than the three sane values listed above."
+
     (let ((found (assoc mac-command-modifier
-                        '((hyper . "H-")
-                          (super . "s-")
-                          (alt   . "A-")))))
-      (when found (kbd (concat (cdr found) keys)))))
+                   '((hyper . "H-")
+                      (super . "s-")
+                      (alt   . "A-")))))
+      (when found (kbd (concat (cdr found) keys))
```

Comment:
The proposed change is easier to read by putting the `when` condition on it's own line:
```elisp
      (when found
        (kbd (concat (cdr found) keys))
        (kbd (concat "H-" keys)))))
```
Only the last line will be returned by the `kbd-mac-command` function. Making the first `kbd` line obsolete.

Were you intending to do this?
```elisp
      (when found (kbd (concat "H-" keys)))))
```

syl20bnr said:
>Instead of hardcoding -H maybe we could set the keys to osx-option-as with the same mechanic so we support other key bindings under osx-option-as or maybe this use-case does not make sense ?
If it does not make sense then I guess we could create a layer variable osx-swap-option-and-commands and document what it does in the README.

Therefore it probably shouldn't be hard coded.

Reply:
It was a long time ago when I setup the config. When `mac-command-modifier` is not one of `hyper` `super` or `alt`, it ended up having not bindings at all. 

Probably we shouldn't hard code this part, or we at least provide some default value here?

Reply:
hi @duianto, thanks for your time reviewing the PR. Really appreciate it. I fixed most of the comments while having one left. 

Actually I was indenting to do give it a default value, i changed `when` to `if`

Reply:
I don't have a mac so i can't test it, these suggestions were just what I noticed.
Oh yeah, now I see it, `if` instead of `when` seems like the way to go. 👍